src/components/CreateAccount.js

```diff
@@ -101,20 +101,28 @@ class CreateAccount extends Component {
             if (this.isWhistle()) {
                 window.postMessage('whistle_signup_complete');
             } else {
-                const urlParameters =
-                    query &&
-                    Object.keys(query)
-                        .map(param => `${param}=${query[param]}`)
-                        .join('&');
                 setTimeout(() => {
-                    window.location.href = `${
-                        window.config.DEFAULT_REDIRECT_URI
-                    }?${urlParameters}`;
+                    window.location.href = `${this.getRedirect()}`;
                 }, 5000);
             }
         }
     }
+    getRedirect = () => {
+        const { username } = this.state;
+        const { location: { query } } = this.props;
 
+        const redirectUri = `${
+            window.config.DEFAULT_REDIRECT_URI
+        }/login.html#account=${username}`;
```

Comment:
This should not be hardcoded, see spec in https://github.com/steemit/faucet/issues/188